src/qt/bitcoingui.cpp

```diff
@@ -1395,6 +1395,14 @@ void BitcoinGUI::showModalOverlay()
         modalOverlay->toggleVisibility();
 }
 
+void BitcoinGUI::clearGUI()
+{
+    trayIconMenu->clear();
```

Comment:
Why clear here?

Reply:
The `trayIconMenu` contains some `QAction` instances. I think we do not need any chance to activate any of them during shutdown.

Reply:
How about `trayIconMenu->setEnabled(false)` instead?